log/logger.go

```diff
@@ -0,0 +1,16 @@
+package log
```

Comment:
Does this need to be a separate package?

Reply:
hmm, the reason this is separate is that it is initiated in main.go but referenced and used on other packages (/kube /calico) and found that way preferable to passing a variable to those in order to use the same Logger. Also, this is copied over kube-service-mirror, so I didn't spend much time on it, I could review if needed?